src/doc/man/cargo-run.md

```diff
@@ -17,6 +17,11 @@ All the arguments following the two dashes (`--`) are passed to the binary to
 run. If you're passing arguments to both Cargo and the binary, the ones after
 `--` go to the binary, the ones before go to Cargo.
 
+Unlike [tests](cargo-test.html#working-directory-of-tests) and 
+[benchmarks](cargo-bench.html#working-directory-of-benchmarks), the working 
+directory of the binary executed with `cargo run` is set to the current working 
+directory, same as if it was executed in the shell directly.
```

Comment:
The text is great! Links in manpage aren't displayed good enough. Do you feel that it is better to remove links here? I am not sure because these links are also very helpful.

<img width="1207" alt="image" src="https://user-images.githubusercontent.com/14314532/228415967-a10aee10-3807-4194-ab96-8fb094ab73a9.png">

Maybe this?

```suggestion
Unlike {{man "cargo-test" 1}} and {{man "cargo-bench" 1}} , the working 
directory of the binary executed with `cargo run` is set to the current working 
directory, same as if it was executed in the shell directly.
```